database/migrations/2022_06_12_075709_add_token_to_user_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use App\Models\Configs;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class AddTokenToUserTable extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 */
+	public function up(): void
+	{
+		Configs::where('key', '=', 'api_key')->delete();
+
+		Schema::table('users', function (Blueprint $table) {
+			$table->char('token', 100)->after('email')->default('');
```

Comment:
Why `''` as default? The column is nullable hence we should use `null` as default or make the column non-nullable. 

But using `null` also yields another advantage. The tokens are unique with very high likelihood. Hence, we should create a `unique` index on this column which also gives higher efficiency when users are looked up by their token. However, a unique column may have multiple `null` entries, because `null` is not included in the index. But SQL does not allow multiple `''` in a unique index.